service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -68,16 +69,16 @@
 @WebMvcTest(CaseManagementOrderProgressionController.class)
 @OverrideAutoConfiguration(enabled = true)
 class CaseManagementOrderProgressionControllerTest extends AbstractControllerTest {
-    private static final UUID uuid = randomUUID();
+    private static final UUID ID = randomUUID();
```

Comment:
I would call this `HEARING_ID` just to give a bit more context as to what the uuid is being used for. More indicates an issue with the CMO class though in my opinion.